partcad/tests/conftest.py

```
@@ -5,3 +5,9 @@
 @pytest.fixture(autouse=True)
 def setup_function() -> None:
     reset_errors()
+
+@pytest.hookimpl()
+def pytest_sessionfinish(session, exitstatus):
```

Ruff (0.8.2)

Missing return type annotation for public function `pytest_sessionfinish`

Add return type annotation: `None`

(ANN201)